scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/RunningTaskData.java

```diff
@@ -44,7 +44,11 @@
 
     private final NodeSet nodes;
 
-    private int pingAttempts = 0;
+    private volatile int pingAttempts = 0;
+
+    private volatile boolean isRestarting = false;
```

Comment:
I do not insist but I thought it is kinda bad practive to start name of boolean with "is".

Reply:
ok, I can rename the variable.